types/src/era_id.rs

```diff
@@ -53,6 +53,15 @@ impl EraId {
         (current_era_id..=current_era_id + num_eras).map(EraId)
     }
 
+    /// Returns an iterator over a range of era IDs, starting from `start` and ending at `end`,
+    /// inclusive.
+    pub fn iter_range_inclusive(
```

Comment:
Since this is basically a helper function, should we be adding it to a public facing crate?

Reply:
Good point. Although we already have similar helper functions above, so it kind of fits here.